run.py

```diff
@@ -52,6 +55,6 @@ def parse_args():
 elif args.feature == 2:
     pass # TODO call second analysis
 elif args.feature == 3:
-    pass # TODO call third analysis
+    TriageTimeAnalysis().triage_time_analysis(show_plot=args.show_plot)
```

Comment:
To match the style of the others, let's make it look like this:
```suggestion
    TriageTimeAnalysis().run()
```

Reply:
Agreed with the observation. Made this change in code.